BLUESPAWN-server/BLUESPAWN-server/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BLUESPAWN_server
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4994516)